content/doc/upgrade-guide/2.60.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: documentation
+title:  Jenkins Upgrade Guide
+notitle: true
+---
+
+== Upgrading to Jenkins LTS 2.60.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.60.1 covers the upgrade from 2.46.3.
+
+=== Upgrading to Jenkins LTS 2.60.1
+
+==== Java 8 Required
+
+Jenkins 2.60.1 is the first LTS release to require Java 8.
```

Comment:
Should the word "require" be bold?  I was expecting more emphasis on that, since it is a major change from prior releases?